src/ucp/api/ucp.h

```diff
@@ -960,6 +971,25 @@ typedef struct ucp_generic_dt_ops {
 } ucp_generic_dt_ops_t;
 
 
+/**
+ * @ingroup UCP_DATATYPE
+ * @brief UCP datatype attributes
+ *
+ * This structure provides attributes that can be queried for a UCP datatype.
+ */
+typedef struct {
+    /**
+     * Mask of valid fields in this structure, using bits from @ref ucp_dt_attr_field.
+     * Fields not specified in this mask will be ignored.
+     * Provides ABI compatibility with respect to adding new fields.
+     */
+    uint64_t field_mask;
+
+     /* Packed size of a single element of the given datatype. */
+     size_t packed_size;
```

Comment:
wrong alignment

Reply:
Fixed. (Unfortunately, the clang-format from UCX's clang fork does not work on my system. It complains about many lines in _.clang-format_.)